raster/src/main/scala/geotrellis/raster/Tile.scala

```diff
@@ -292,4 +292,10 @@ trait Tile extends CellGrid with IterableTile with MappableTile[Tile] {
     }
     s
   }
+
+  def isNoDataTile = {
+    var empty = true
+    foreach { z => if (isData(z)) empty = false }
```

Comment:
@lossyrob ok will make a probably small pr, yes throwing an exception can be significant :/